tags/resources/tag-formation-templates/template-roles.md

```diff
@@ -1,3 +1,103 @@
+* Organize, host, plan and facilitate TAG meetings and events.
+* Reporting to the CNCF TOC on the status of TAG work.
+* Encouraging community members to start projects and working groups.
+* Resolve technical difficulties and decisions related to multiple sub-projects.
+* Remind the community of the scope of the TAG and point out guardrails in discussions, working groups, and projects.
+* Serve as TAG leadership representative in CNCF project discussions and at community events.
+* Evolve the TAG to reflect ongoing changes in the industry.
+* Onboard and mentor new community members.
+* Mentor community members in a leadership role within the TAG.
+* Enforce and promote diversity in TAG work.
+* Enforce and promote good communication in TAG efforts in accordance with the [CNCF CoC](https://www.cncf.io/conduct/)
+
+## Technical lead
+
+A technical lead (TL) expands the bandwidth of the leadership team in terms of covering technical areas, growing the community, organizing events or doing TAG communications.
+Proposals must have a TL or Chair working as an active sponsor.
+TLs are experienced contributors within the CNCF community with rooted expertise in the TAG area. TL serve a two-year term but may renew their term by submitting a follow-up application.
+Technical leads go through an [election process](https://github.com/cncf/toc/blob/master/tags/cncf-tags.md#elections) with a final CNCF TOC vote.
+The [leadership election process is described here](template-leadership-election-process.md#technical-lead).
+
+**A technical lead is expected to**:
+* Establish and guide projects and working groups.
+* Resolve cross-sub-project technical difficulties and decisions.
+* Propose agenda items for meetings to ensure that open issues are discussed with the group when needed.
+* Serving as TAG leadership representative in CNCF project discussions and on community events.
+* Onboard and mentor new community members.
+* Mentor community members in a leadership role within the TAG.
+* Enforce and encourage diversity in the TAG efforts.
+* Enforce and encourage good communication in the TAG efforts following the [CNCF CoC](https://www.cncf.io/conduct/).
+* Support the TAG Chairs.
+
+## Working Group Lead
+
+A working group lead is an experienced contributor to the TAG that applies to lead along one or two others a working group.
+Working groups as community governance structure are defined in the [CNCF TAG](https://github.com/cncf/toc/blob/master/tags) folder.
+TAG Chairs are responsible facilitating the working group applications and make sure that the process is done transparent, accessible and traceable for everyone.
+One of the TAG chairs serves as the working group sponsor and provides guidance to the working group leads.
+Working group leads are signed off by the TAG Chairs and the TOC Liaisons.
+The [leadership election process is described here](template-leadership-election-process.md#working-group-lead).
+
+**A working group lead is expected to**:
```

Comment:
I think 'expected' is a strong word (as in being evaluated in a performance review and that all these responsibilities are needed for a WG tech lead). Imo, in open source we'd like to encourage people to step up (not scare away), be excited, etc, even if they are doing this work part-time/pro bono.

Suggestion:

**A working group lead responsibilities can include the following**:

Reply:
👍, added